### PR TITLE
Add styling for member link

### PR DESCRIPTION
**Pull request by mattandrews**
This adds some design requests to the (currently off) member signup link.

It looks like this on large displays:

screen shot 2015-05-06 at 16 51 03

... and for smaller ones we hide the link text and button ("free" on its own looks a bit odd):

screen shot 2015-05-06 at 16 51 34

I've also extended the feature for another month (that okay with you, @crifmulholland?).

Have at it!

### CODE REVIEW

**Comment by mattandrews** [comment]
cc @davidrapson

File: static/src/stylesheets/module/nav/_control.scss [hunk 1]
```diff
@@ -40,6 +40,10 @@ $control-size: 36px;
     }
 }
 
+.control__info--supporting {
```
Comment (davidrapson) [inline static/src/stylesheets/module/nav/_control.scss]: Always felt weird that there's a `.control__info` but no .`control` class in these styles… You're following the rest of the file though, so it's fine.

**Comment by mattandrews** [comment]
@sndrs would you mind giving this a look? Getting chased to get this out ASAP...

**Comment by davidrapson** [comment]
Didn't realise this was going to be _as well as_ the sign in link. Double user icon looks a little odd on the small screen version.

:+1: For the code though.

**Comment by crifmulholland** [comment]
thanks matt. Does the new icon completely disappear at the smallest breakpoint? I don't think there's enough room for it at 320 px, so it should do.

**Comment by mattandrews** [comment]
@crifmulholland no, it stays there (no "free" or other text). It looks okay at that size, though. Want it gone?

**Comment by crifmulholland** [comment]
Yeah, it wouldn't fit on iphone 5. Also, as just mentioned, can we scope it to UK edition only please!

Cheers, C

**Comment by mattandrews** [comment]
Was actually wrong; it *does* disappear for mobile breakpoints:

screen shot 2015-05-07 at 14 23 42

Just added a check for the UK edition scoping:

screen shot 2015-05-07 at 14 23 17
screen shot 2015-05-07 at 14 23 30

@OliverJAsh, are you okay to give this a quick code review? Needs to go out ASAP if possible.

**Comment by OliverJAsh** [comment]
Nice! :+1: Cheers for doing this. Sorry I didn't get round to replying sooner.

**Comment by mattandrews** [comment]
Had to merge in https://github.com/guardian/frontend/pull/9122 to fix the build, will deploy shortly.

**Pull request merged**